### PR TITLE
Repair draw_deps in Makefile

### PR DESCRIPTION
**Pull request by adrianbrink**
Sorry for the merge commit. Next time I'll rebase :-)

### CODE REVIEW

**Comment by rigelrozanski** [comment]
Cool dep. graphic!

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ dist:
 
 test:
 	@echo "--> Running go test"
-	@go test $(PACKAGES)
+	@go test -v $(PACKAGES)
```
Comment (ebuchman) [inline Makefile]: I prefer to not have this because there's too much output when you just want to see a bunch of "ok"

File: README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/6874
 )](https://godoc.org/github.com/tendermint/tendermint)
 [![chat](https://img.shields.io/badge/slack-join%20chat-pink.svg)](http://forum.tendermint.com:3000/)
 [![license](https://img.shields.io/github/license/tendermint/tendermint.svg)](https://github.com/tendermint/tendermint/blob/master/LICENSE)
+[![](https://tokei.rs/b1/github/tendermint/tendermint?category=lines)](https://github.com/tendermint/tendermint)
```
Comment (ebuchman) [inline README.md]: love it!

**Comment by ebuchman** [comment]
Adrian - this is a wonderful little PR. Would you mind removing the `-v` in `make test` and squashing this all down to one commit?

Thanks!

**Comment by ebuchman** [comment]
Oh actually nevermind the squash. two commits here is fine. But please remove the `-v` and rebase on develop :)

**Comment by ebuchman** [comment]
also, let's put the png under docs rather than in the root!

**Comment by adrianbrink** [comment]
Okay, I'll do that tomorrow. :-)

**Comment by melekes** [comment]
> also, let's put the png under docs rather than in the root!

I'd prefer not to store it because it will constantly be changing.

**Comment by ebuchman** [comment]
cherry picked commits and cleaned up in https://github.com/tendermint/tendermint/pull/443

**Pull request closed without merging**